File: C# + .NET Core + ASP.NET/Models/Projection.cs
using System.Text;
using apples.Models;

namespace apples.Models
{
    public class Projection
    {
        private StringBuilder State { get; } = new StringBuilder();

        public void Apply(Event evt)
        {
            State.Append(evt.Data);
        }

        public string GetState()
        {
            return State.ToString();
        }
    }
}

File: Python + CPython + Flask/Projection.py
from Event import Event

class Projection:

    def __init__(self):
        self.state = ""

    def apply(self, event):
        self.state += event.data

    def load_from_events(self, aggregate_uuid):
        events = Event.query.filter_by(aggregate_uuid=aggregate_uuid).all()
        for event in events:
            self.apply(event)

    def get_state(self):
        return self.state